CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+### Before you Start
+- Unless stated in README.md. Make sure you are pulling the source code from main branch with recursive variable to pull all submodules Collapse need to run.
+
+    ```git pull https://github.com/neon-nyan/Collapse --recursive```
+    
+- Make sure you use the supported IDE/SDKs listed below.
+- Make sure to read the Feature(s) restriction below.

```

Comment:
Change to: Make sure to read the "Restriction for New Feature(s)" section of this document, below.

Reply:
Applied